apis/apiextensions/v1alpha1/usage_types.go

```diff
@@ -68,6 +68,9 @@ type UsageSpec struct {
 	// Reason is the reason for blocking deletion of the resource.
 	// +optional
 	Reason *string `json:"reason,omitempty"`
+	// ReplayDeletion will trigger a deletion on the used resource during the deletion of the usage itself, if it was attempted to be deleted at least once.
+	// +optional
+	ReplayDeletion *bool `json:"replayDeletion,omitempty"`
```

Comment:
Would it make sense to default this to `true`?  That would provide the best user experience.  I don't know why someone wouldn't want this to happen, but it's good to have the option to disable it.

Reply:
I would prefer to start with `false,` and consider enabling it by default while bumping the API to beta, where we could have more freedom to make behavioral changes.

Reply:
Added as a todo item to the promote to beta ticket: https://github.com/crossplane/crossplane/issues/4622